sixMans/game.py

```diff
@@ -282,8 +556,54 @@ def _get_pickable_players_str(self):
             players += "{} {}\n".format(react, player.mention)
         return players
 
-    async def pick_balanced_teams(self):
-        pass
+    async def update_game_info(self, helper_role=None, invalid=False, prefix='?'):
+        if not helper_role:
+            helper_role = self.helper_role
+        sm_title = "{0} {1} Mans Game Info".format(self.queue.name, self.queue.queueMaxSize)
+        embed_color = discord.Colour.green()
+        if invalid:
+            sm_title += " :x: [Teams Changed]"
+            embed_color = discord.Colour.red()
+        embed = discord.Embed(title=sm_title, color=embed_color)
+        embed.set_thumbnail(url=self.queue.guild.icon_url)
+
+        if self.queue.teamSelection == Strings.VOTE_TS:
```

Comment:
Is there a reason to look at the queue.teamSelection here instead of self.teamSelection?

Reply:
Yes. When the team selection is "vote," the game info embed shows the team selection method that won the vote. If the players voted for random, it shows "🎲 Random." If balanced won the vote, "☯️ Balanced (beta)" will be shown. If the queue is set to random, then it won't show that in the game info embed.

So it looks at the queue team selection to determine whether or not to show the team selection mode. Then it looks at the game (self) team selection mode to print if it was voted for.